src/main/src/main/java/org/geoserver/config/impl/ContactInfoImpl.java

```diff
@@ -39,6 +43,34 @@
 
     String onlineResource;
 
+    InternationalString internationalAddress;
```

Comment:
Do these really need to be package visible? Can they be `private`?

Reply:
fixed